lms/djangoapps/courseware/tests/test_module_render.py

```diff
@@ -692,25 +691,22 @@ def test_grade_event_with_completion_disabled(self):
 
     @XBlock.register_temp_plugin(StubCompletableXBlock, identifier='comp')
     def test_grade_event(self):
-        new_features = settings.FEATURES.copy()
-        new_features['MARK_PROGRESS_ON_GRADING_EVENT'] = True
-        with override_settings(FEATURES=new_features):
```

Comment:
We no longer need to enable this flag, since the alternative API automatically enables this.